server/src/core/payment/transaction-service.ts

```diff
@@ -87,6 +90,7 @@ export class Transactions implements TransactionService {
   }
 
   async sendDonation(from: User, to: User, args: SendDonationArgs): Promise<Transaction> {
+    validators.validatesSendDonation({ from: from._id, to: to._id, amount: args.amount });
```

Comment:
Ideally we should throw a validation error if the `args` contain other fields from what's allowed. But this is not being validated here.